porchlightapi/settings.py

```diff
@@ -35,5 +36,14 @@
                                'PORCHLIGHT_JSON_FILE',
                                PORCHLIGHT_JSON_FILE_DEFAULT)
 
+PORCHLIGHT_GITHUB_AUTH_DEFAULT = None
+PORCHLIGHT_GITHUB_AUTH = getattr(settings,
+                                 'PORCHLIGHT_GITHUB_AUTH',
+                                 PORCHLIGHT_GITHUB_AUTH_DEFAULT)
+
+PORCHLIGHT_GITHUB_TAG_PATTERN_DEFAULT = r'^v[.0-9]+'
```

Comment:
@Scotchester I can change it, but it's just a default value there. We can override it for our own installation's `local_settings.py` (in fact, it's a setting I'd like to be overridable on a per-repo basis but that's not something I have a mechanism in place for supporting). I just think that a more descriptive default value is better, but for reasons that could easily be accomplished in documentation as well.